package.json

```diff
@@ -35,6 +35,7 @@
     "react-dom": "^16.5.2",
     "react-ga": "^2.5.3",
     "react-helmet": "^5.2.0",
+    "react-icons": "^3.2.1",
```

Comment:
Could you also include the types for react-icons? https://www.npmjs.com/package/@types/react-icons